plugins/woocommerce/src/Internal/DataStores/StockNotifications/StockNotificationsDataStore.php

```diff
@@ -0,0 +1,532 @@
+<?php
+/**
+ * StockNotificationsDataStore class file.
+ */
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Internal\DataStores\StockNotifications;
+
+use Automattic\WooCommerce\Internal\StockNotifications\Notification;
+use Automattic\WooCommerce\Internal\Utilities\DatabaseUtil;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * The Stock Notifications Data Store.
+ */
+class StockNotificationsDataStore implements \WC_Object_Data_Store_Interface {
+
+	/**
+	 * The database util object to use.
+	 *
+	 * @var DatabaseUtil
+	 */
+	protected $database_util;
```

Comment:
You can add type declarations for better type safety.

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/57062/commits/5c6c0133702a90ef6f29a36a7e6198e668b43a16